src/components/shared/AlertBox.tsx

```diff
@@ -0,0 +1,119 @@
+import { FormattedMessage } from 'react-intl';
+import { BaseComponentProps } from 'types';
+
+interface Props extends BaseComponentProps {
+    severity: AlertColor;
+    short?: boolean;
+    hideIcon?: boolean;
+    title?: string | ReactNode;
+}
+
+const SHARED_STYLING = {
+    borderRadius: 4,
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
+};
+
+const HEADER_MESSAGE = {
+    warning: 'alert.warning',
+    success: 'alert.success',
+    info: 'alert.info',
+    error: 'alert.error',
+};
+
+function AlertBox({ short, severity, hideIcon, title, children }: Props) {
+    const iconComponentStyling = useMemo(
+        () =>
+            !short
+                ? {
+                      color: (theme: Theme) =>
+                          alertTextPrimary[theme.palette.mode],
+                      fontSize: '2em',
+                      marginLeft: 'auto',
+                      marginRight: 'auto',
+                  }
+                : undefined,
+        [short]
+    );
+
+    const header = useMemo(
+        () =>
+            !short && HEADER_MESSAGE[severity] ? (
+                <Typography
+                    variant="h4"
+                    component="div"
+                    sx={{
+                        pb: 1,
+                    }}
+                >
+                    <FormattedMessage id={HEADER_MESSAGE[severity]} />
+                </Typography>
+            ) : null,
+        [severity, short]
+    );
+
+    return (
+        <Alert
+            severity={severity}
+            variant="outlined"
+            icon={hideIcon ?? undefined}
+            iconMapping={{
+                error: <DangerousOutlinedIcon sx={iconComponentStyling} />,
+                warning: (
+                    <ReportProblemOutlinedIcon sx={iconComponentStyling} />
+                ),
+                info: <InfoOutlinedIcon sx={iconComponentStyling} />,
+                success: <TaskAltOutlinedIcon sx={iconComponentStyling} />,
+            }}
+            sx={{
+                'backgroundColor': (theme) =>
+                    alertBackground[theme.palette.mode],
+                'color': (theme) => alertTextPrimary[theme.palette.mode],
+                'borderColor': (theme) =>
+                    theme.palette[severity][theme.palette.mode],
+                'padding': 0,
+                '& > .MuiAlert-message': {
+                    p: 1,
+                },
+                '& > .MuiAlert-icon': short
+                    ? {
+                          ...SHARED_STYLING,
+                          borderLeftColor: (theme) =>
+                              theme.palette[severity][theme.palette.mode],
+                          color: (theme) =>
+                              theme.palette[severity][theme.palette.mode],
+                          mr: 0,
+                          px: 1,
+                          borderLeftStyle: 'solid',
+                          borderLeftWidth: 15,
```

Comment:
Short does not include the background color behind the icon and just adds a big thick border on the left.